### ISSUE TITLE
Include nmos-multi-node?

### ISSUE THREAD

**Issue by garethsb** [rhastie/build-nmos-cpp#26]
Would be nice to investigate how much baggage including https://github.com/garethsb/nmos-cpp-examples/tree/main/nmos-multi-node would add to the container.

**Comment by rhastie** [comment]
We can potentially add it by extending the "RUN_NODE" functionality in the entrypoint.sh script. It will however increase the container size quite considerably (at least double its size) as we don't currently install or use Python 3 in the runtime container. We might consider moving from Ubuntu as the runtime container base to the python-slim base which is built from Debian. If we do I would feel more comfortable moving the entire Dockerfile to Debian

**Comment by garethsb** [comment]
Yeah that was my concern... maybe it's an extra layer / different tag, or maybe this is a silly idea 🙂
